### ISSUE TITLE
Deleting a document should name the document to be deleted

### ISSUE THREAD

**Issue by sopoforic** [burtonator/polar-bookshelf#1586]
Right now (2.0.114), deleting a document merely gives a confirmation modal that reads:

> Are you sure you want to delete these item(s)?  
> This is a permanent operation and can't be undone.

This box should name the document to be deleted, e.g.

> Are you sure you want to delete these item(s)?  
> **Alice's Adventures in Wonderland** will be deleted. This is a permanent operation and can't be undone.

**Comment by burtonator** [comment]
good point... we're going to do another spring on lots of reader usability bugs and I'll add this to that ...
